### ISSUE TITLE
Unified settings interface

### ISSUE THREAD

**Issue by akaleeroy** [facelessuser/MarkdownPreview#6]
Sublime Text itself and [SublimeLinter](https://packagecontrol.io/packages/SublimeLinter) implement this side-by-side interface for configuring settings. It's very convenient, try adding it!

**Comment by facelessuser** [comment]
Yeah, a lot of my packages use it, I just haven't converted this one over to do that yet.  I will address this in a future release.
